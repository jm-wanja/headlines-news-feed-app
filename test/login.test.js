import React from 'react';
import { mount, shallow } from 'enzyme';
import shallowToJSON from 'enzyme-to-json';
import GooglePlusLogin from '../client/components/headlinesJs/LoginButton.js';

describe('login button component', () => {
    it('snapshot of GooglePlusLogin', () => {
        const login = shallow(<GooglePlusLogin />);
        const tree = shallowToJSON(login);
        expect(tree).toMatchSnapshot();
    })

    it('asserts there is a googlplus button', () => {
        const login = shallow(<GooglePlusLogin />);
        expect(login.find('#login')).toBe.defined;
    })
})