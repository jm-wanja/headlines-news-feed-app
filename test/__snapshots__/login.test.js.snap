// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login button component snapshot of GooglePlusLogin 1`] = `
<div
  id="login"
>
  <Button
    accent={true}
    raised={true}
    ripple={true}
  >
    GooglePlusLogin
  </Button>
</div>
`;
